cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -215,10 +221,16 @@ func (t *GceTemplateBuilder) BuildNodeFromTemplate(mig Mig, template *gce.Instan
 		return nil, fmt.Errorf("could not fetch ephemeral storage from instance template: %v", err)
 	}
 
-	capacity, err := t.BuildCapacity(cpu, mem, template.Properties.GuestAccelerators, os, osDistribution, arch, ephemeralStorage, ephemeralStorageLocalSsdCount, pods, mig.Version(), reserved)
+	extendedResources, err := extractExtendedResourcesFromKubeEnv(kubeEnvValue)
+	if err != nil {
+		return nil, fmt.Errorf("could not fetch extended resources from instance template: %v", err)
```

Comment:
I don't think this should break TemplateNode building. Perhaps just logging an error would be sufficient? Having a TemplateNode with just the basic resources sounds better than not having it at all due to a typo on in `extended_resources` variable definition.

Reply:
Sounds legit 👍 I'll make the change.